### PR TITLE
Add support for v2 schema

### PR DESCRIPTION
**Pull request by averyniceday**
Using existing SV model for majority of changes
Add a check for Gene 1 (required not null on CVR end) to determine if v1/v2
Add site1_exon/site2_exon support

### CODE REVIEW

File: cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/model/CVRSvVariant.java [hunk 1]
```diff
@@ -159,36 +165,43 @@ public CVRSvVariant() {
...
-    * @param mapq
-    * @param variantStatusName
-    * @param site1Desc
-    * @param comments
-    * @param site2Chrom
-    * @param site2Gene
-    * @param annotation
-    */
-    public CVRSvVariant(String annotation, String breakpointType, String comments, String confidenceClass, String connType, String connectionType, String eventInfo, String mapq, String normalReadCount, String normalVariantCount, String pairedEndReadSupport, String site1Chrom, String site1Desc, String site1Gene, String site1Pos, String site2Chrom, String site2Desc, String site2Gene, String site2Pos, String splitReadSupport, String svClassName, String svDesc, String svLength, String svVariantId, String tumorReadCount, String tumorVariantCount, String variantStatusName) {
+     * @param annotation
```
Comment (averyniceday) [inline cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/model/CVRSvVariant.java:168]: this was auto-generated from IntelliJ, I don't think it hurts to update it? don't think it's being used anywhere

File: cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/model/staging/CVRSvRecord.java [hunk 2]
```diff
@@ -105,7 +109,61 @@ public CVRSvRecord(CVRSvVariant variant, String sampleId) {
         this.tumor_read_count = variant.getTumor_Read_Count();
         this.tumor_variant_count = variant.getTumor_Variant_Count();
         this.variant_status_name = variant.getVariant_Status_Name();
+
+        // CVR confirmed Site1_GENE/Gene1 is a NOT_NULL field
+        // Use Gene1 to test whether v1 or v2 schema
+        if (variant.getSite1_Gene() != null && !variant.getSite1_Gene().isEmpty()) {
+            this.site1_gene = variant.getSite1_Gene();
```
Comment (mandawilson) [inline cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/model/staging/CVRSvRecord.java:116]: Delete this.site1_gene and this.site2_gene initialisation above.

Comment (mandawilson) [reply cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/model/staging/CVRSvRecord.java:116]: Check for calls to setGene1 and setGene2 to see if site1_gene and site2_gene needs setting in that case.

Comment (mandawilson) [reply cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/model/staging/CVRSvRecord.java:113]: Avery looked it over and said these are only called once and only for the data_sv.txt not the cvr json file (so this is correct behaviour).

**Pull request merged**